### ISSUE TITLE
Video specific SEO features directly from Wordpress Plugin

### ISSUE THREAD

**Issue by gittyuphub** [s3bubble/Streamium-Netflix-Theme#101]
## Description
S3Bubble does not have support for google structured video data with the wordpress plugin.
The theme may have options to install SEO plugins, but this does not address structured SEO data for specifically video.

S3Bubble should be compatible with the Yoast Video SEO plugin, or provide native Video SEO features so that video content embedded on our webpages is ingested by google as a "video" and will display in google search results with the video thumbnail and description.

[https://kb.yoast.com/kb/supported-plugins-of-the-video-seo-plugin/](url)
[](url)

Such as this link.

Or.... have you considered being compatible with Yoast's SEO video plugin?

Reason is, I'm looking for my site videos to show up as a google video search result with a thumbnail. I can do this with the Yoast Video SEO plugin along with Vimeo (both are compatible together) 

But, I've been looking for an S3 option, and your plugin is pretty much exactly what I'm looking for!

I considered S3 Media Maestro (supports cloudfront) but it's missing Elastic transcoder, variable bitrate/resolution etc.

....just wondering about the SEO options you have in S3 Bubble.

thanks
## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1.
2.
3.

### Expected
Please describe what you expected to see.

### Actual
Please describe what actually happened.

### Error output
If there are any errors at all, please include them here.

## Additional Information
Please include any additional information necessary here. Including the following:

#### Streamium Theme
what version of the theme are you using?
#### browsers
what browser are affected?
#### OSes
what platforms (operating systems and devices) are affected?

**Comment by ChorchMecanica** [comment]
Yes, we need this too. Our site is 100% video content, that is way we choose streamium to work with, but the lak of compatibility with structured data was an unpleacent surprise :(
